documentation/cli-usage.md

```diff
@@ -0,0 +1,46 @@
+<small>*[documentation](/README.md#documentation) / kyverno-cli*</small>
+
+
+# [kyverno CLI](https://github.com/nirmata/kyverno/releases) - kubectl plugin to deal with kyverno policies
+Used to validate policies and apply policies on resources. 
```

Comment:
Change `Used to validate policies and apply policies on resources.` to:

The Kyverno Command Line Interface (CLI) is designed to validate policies and test the behavior of applying policies to resources before adding the policy to a cluster. It can be used as a kubectl plugin and as a standalone CLI.